editors/code/src/tasks.ts

```diff
@@ -145,11 +145,13 @@ async function cargoToExecution(
         const cargoPath = await toolchain.cargoPath();
         const cargoCommand = definition.overrideCargo?.split(" ") ?? [cargoPath];
 
-        const args = [definition.command].concat(definition.args ?? []);
-        const fullCommand = [...cargoCommand, ...args];
-        const processName = unwrapUndefinable(fullCommand[0]);
+        const args = expectNotUndefined(
+            definition.args,
```

Comment:
I'm not really familiar with the code here, but shouldn't we use `definition.command` just like before?

Reply:
i'm not sure what you mean—can you expand?

Reply:
We're not using `definition.command` in the `cargo` case. I guess it's also present in `definition.args`, then?

Reply:
Shouldn't we drop the entire `cargoCommand` querying here? If I look at the previous PR, `createTask` already handles this now no?

Reply:
uh... yes. i think we can drop it.